cli/userstatus.go

```diff
@@ -74,7 +71,7 @@ func createUserStatusCommand(sdkStatus codersdk.UserStatus) *cobra.Command {
 			_, err = cliui.Prompt(cmd, cliui.PromptOptions{
 				Text:      fmt.Sprintf("Are you sure you want to %s this user?", verb),
 				IsConfirm: true,
-				Default:   defaultConfirm,
+				Default:   cliui.ConfirmYes,
```

Comment:
Yes, as per the discussion on the issue, the requirements were updated. We want to default to `no` only in case of deleting templates and workspaces -- the non-reversible kind of deletions.

Reply:
Ah, got it, I just missed seeing your edit on the PR description. LGTM!